### PR TITLE
fix: upgrade to substrate 0.9.11

### PR DESCRIPTION
**Pull request by sander2**

### CODE REVIEW

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -7,11 +7,10 @@ edition = "2018"
 [features]
 default = []
 testing-utils = [
-    "substrate-subxt/client",
-    "substrate-subxt-client",
+    # "substrate-subxt/client",
+    # "substrate-subxt-client",
```
[the remark this answers is not in the document]
Comment (sander2) [reply runtime/Cargo.toml:11]: for now in our fork the client is compiled unconditionally. I'm going to submit a pr to get the client back in upstream, but we shouldn't block merging this pr for that

File: runtime/src/error.rs [hunk 2]
```diff
@@ -70,38 +67,42 @@ pub enum Error {
...
     pub fn is_duplicate_block(&self) -> bool {
         matches!(self,
-            Error::SubxtError(SubxtError::Runtime(SubxtRuntimeError::Module(SubxtModuleError {
-                ref module,
+            Error::SubxtError(SubxtError::Runtime(SubxtRuntimeError::Module(SubxtPalletError {
+                ref pallet,
                 ref error,
-            }))) if module == BTC_RELAY_MODULE && error == DUPLICATE_BLOCK_ERROR
+                ..
+            }))) if pallet == BTC_RELAY_MODULE && error == &format!("{:?}", BtcRelayPalletError::DuplicateBlock)
```
[the remark this answers is not in the document]
Comment (sander2) [reply runtime/src/error.rs:74]: `BtcRelayPalletError` is an enum, with `DuplicateBlock` being one of the variants. The `format!` is a bit of a hack, but it ensures we at least use the correct name - if the field name ever changes, we will get a compile error

File: runtime/src/lib.rs [hunk 3]
```diff
@@ -1,86 +1,84 @@
...
-use xcm::v0::{Error as XcmError, NetworkId};
+pub const RELAY_CHAIN_CURRENCY: CurrencyId = CurrencyId::DOT;
+pub const RELAY_CHAIN_WRAPPED_CURRENCY: CurrencyId = CurrencyId::KBTC;
 
-pub const TX_FEES: u128 = 2000000000;
+pub const BTC_RELAY_MODULE: &str = "BTCRelay";
+pub const ISSUE_MODULE: &str = "Issue";
+pub const REDEEM_MODULE: &str = "Redeem";
+pub const RELAY_MODULE: &str = "Relay";
+pub const SECURITY_MODULE: &str = "Security";
```
[the remark this answers is not in the document]
Comment (sander2) [reply runtime/src/lib.rs:45]: It is included as part of specific calls/events/storage items, but that's it. We could use a random call and use that, but for now I think this is fine. Long-term we may want to modify the codegen in subxt to allow for easier error matching

File: procedural/src/lib.rs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::{ItemFn, Result};
+
+// todo: This can probably be removed when we upgrade to tokio 1.13 (we need https://github.com/tokio-rs/tokio/pull/4203)
```
[the remark this answers is not in the document]
Comment (sander2) [reply procedural/src/lib.rs:5]: done

File: runtime/src/lib.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (sander2) [reply runtime/src/lib.rs:45]: https://github.com/paritytech/subxt/issues/313

**Review by gregdhill — APPROVED** [review]

**Pull request merged**